src/_scss/pages/awardV2/awardPage.scss

```diff
@@ -77,9 +77,7 @@
                 }
             }
 
-            @import "./visualizations/overview/overview";
```

Comment:
This import statement should stay in, `@import "./visualizations/overview/dates";` can be removed.